velox/docs/functions/spark/string.rst

```diff
@@ -2,17 +2,8 @@
 String Functions
 ====================================
 
-.. note::
-
-    These functions assume that the input strings contain valid UTF-8 encoded
-    Unicode code points. There are no explicit checks for valid UTF-8 and
-    the functions may return incorrect results on invalid UTF-8.
-
-    Additionally, the functions operate on Unicode code points and not user
-    visible *characters* (or *grapheme clusters*).  Some languages combine
-    multiple code points into a single user-perceived *character*, the basic
-    unit of a writing system for a language, but the functions will treat each
-    code point as a separate unit.
+For functions contains, endsWith, startsWith, returns NULL if either left or right is NULL. 
```

Comment:
Isn't this true for other functions, i.e.ascii and chr? I assume this is generally true, hence, let's say:

```
Unless specified otherwise, all functions return NULL if at least one of the arguments is NULL.
```